packages/composite-checkout-wpcom/package.json

```diff
@@ -46,7 +46,8 @@
 		"i18n-calypso": "file:../i18n-calypso",
 		"prop-types": "^15.7.2",
 		"react": "^16.8.6",
-		"react-stripe-elements": "^5.1.0"
+		"react-stripe-elements": "^5.1.0",
+		"jest-emotion": "^10.0.27"
```

Comment:
`jest-emotion` is a dev dependency and shouldn't be here. Only in the top-level `package.json` and maybe also in `packages/calypso-build/package.json`, which provides the common Jest config for all packages.